libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -15,14 +15,7 @@ import com.mapbox.navigation.base.trip.model.RouteLegProgress
 import com.mapbox.navigation.base.trip.model.RouteProgress
 import com.mapbox.navigation.base.trip.model.roadobject.UpcomingRoadObject
 import com.mapbox.navigation.core.directions.session.RoutesExtra
-import com.mapbox.navigation.core.navigator.getLocationMatcherResult
-import com.mapbox.navigation.core.navigator.getRouteInitInfo
-import com.mapbox.navigation.core.navigator.getRouteProgressFrom
-import com.mapbox.navigation.core.navigator.getTripStatusFrom
-import com.mapbox.navigation.core.navigator.mapToDirectionsApi
-import com.mapbox.navigation.core.navigator.toFixLocation
-import com.mapbox.navigation.core.navigator.toLocation
-import com.mapbox.navigation.core.navigator.toLocations
+import com.mapbox.navigation.core.navigator.*
```

Comment:
wild import is not allowed, ktlint fails

Reply:
yep. trying to setup Android Studio styles right now